CHANGELOG.md

```
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.0.0 (28.01.26)
+- [fix: change execution of scripts, and installs of gems.](https://github.com/kommitters/bas_use_cases/pull/287)
+- [Simplify staging deploy by using a single Docker image and removing build steps](https://github.com/kommitters/bas_use_cases/pull/285)
+- [feat: Remove warehouse-related web server routes](https://github.com/kommitters/bas_use_cases/pull/283)
+- [Cleanup BAS Use Cases by removing Warehouse implementations and updating pipelines ](https://github.com/kommitters/bas_use_cases/pull/282)
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)